docs/source/performance.rst

```diff
@@ -114,3 +114,95 @@ However, know that 16-bit and multi-processing (any DDP) can have issues. Here a
     CUDA_LAUNCH_BLOCKING=1 python main.py
 
 .. tip:: We also recommend using 16-bit native found in PyTorch 1.6. Just install this version and Lightning will automatically use it.
+
```

Comment:
Might want to move this out of the PR so we can focus on getting the feature in? Also allows us to look at the docs in greater detail after.

Reply:
Agree. Removed it.